spidermon/contrib/scrapy/monitors.py

```diff
@@ -374,6 +380,53 @@ def test_check_if_field_coverage_rules_are_met(self):
         self.assertTrue(len(failures) == 0, msg=msg)
 
 
+@monitors.name("Jobs Comparison Monitor")
+class JobsComparisonMonitor(BaseScrapyMonitor):
```

Comment:
**suggestion** You could use the new [BaseStatMonitor](https://spidermon.readthedocs.io/en/latest/monitors.html#base-stat-monitor) to create this monitor and define a custom `get_threshold` method that do all the logic of getting data from previous jobs.

Reply:
Sure thing. Will do that.